features/shared/checkout_previous_branch_after_success/same_current_branch/previous_branch_renamed.feature

```diff
@@ -0,0 +1,11 @@
+Feature: when the previous branch is renamed during a Git Town command, the main branch becomes the new previous branch
```

Comment:
I agree that the best behavior would take us back to the renamed branch.

However, this is an extreme edge case that's actually not even possible right now. `git rename-branch` immediately takes us to the newly-renamed branch. Given the level of work needed to support this, I think we're fine leaving it as-is for now.